js/index.js

```diff
@@ -44,3 +44,34 @@ messageForm.addEventListener("submit", function (event) {
   newMessage.appendChild(removeButton);
   messageList.appendChild(newMessage);
 });
+// AJAX
+// New XMLHttpRequest object
+let githubRequest = new XMLHttpRequest();
+// Open a request
+githubRequest.open("GET", "https://api.github.com/users/SokolovskayaE/repos");
+// Send the request
+githubRequest.send();
+
+// Add a "load" event listener
+githubRequest.addEventListener("load", (event) => {
+  if (githubRequest.status === 200) {
+    // Successful HTTP request
+    let repositories = JSON.parse(githubRequest.response); // Parse the response and store it in a variable named repositories
+    console.log(repositories);
+
+    // Select the #projects section by id and store it in a variable named projectSection
+    let projectSection = document.getElementById("projects");
+    // Query the projectSection (instead of the entire document) to find the <ul> element and store it in a variable named projectList
+    let projectList = projectSection.querySelector("ul");
+    // For loop to iterate over your repositories Array, starting at index 0
+    for (let i = 0; i < repositories.length; i++) {
+      // Create a new list item (li) element and store it in a variable named project
+      let project = document.createElement("li");
+      //project.innerText = repositories[i].name; // Set the inner text of your project variable to the current Array element's name property
+      // Transform your repository names into <a> tags that link to GitHub (hint: html_url property)
+      project.innerHTML = `<a class="link link--no-decor" href="https://github.com//SokolovskayaE/fenrir-intro ">Intro to programming FENRIR</a>`;
```

Comment:
The two repositories look like duplicates when viewed, since the text is hard-coded in the <a> element.  You could use 'description', 'htm_url',  or 'name' in the repository object to give them unique names and avoid hard coding. 
e.g. `repositories[i].name`

Reply:
I deleted the link and left only as I had before: project.innerText = repositories[i].name;
As I understated as what you were suggesting.
Thanks a lot!